code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -43,14 +43,25 @@
 	heldname = sanitize_name(stripped_input(user, "Enter new robot name", "Cyborg Reclassification", heldname, MAX_NAME_LEN), allow_numbers = TRUE)
 	log_game("[key_name(user)] have set \"[heldname]\" as a name in a cyborg reclassification board at [loc_name(user)]")
 
+/**
+ * Renames cyborgs when clicked on with a renaming board.
+ *
+ * Updates the cyborg's real_name, key_name, and custom_name to the choosen name (heldname).
+ * Notify the AI if there is a change.
+ * Note: Shells reset their names on deployment, so renaming them is not as useful as it is for cyborgs.
+ *
+ * Arguments:
+ * * R — a robot: a cyborg or a shell
+ * * user — the character to use the board on the cyborg
+ */
```

Comment:
we dont do documentation on procs that already exist i think

Reply:
Alright, I removed them
Thanks for pointing it out